### PR TITLE
modified simulation.js for additional parameters

### PR DESCRIPTION
**Pull request by crystalycteh**

### CODE REVIEW

File: client/src/stores/simulation.js [hunk 1]
```diff
@@ -34,6 +21,33 @@ export const useSimulationStore = defineStore("simulation", {
       if (this.parameters.mtdInterval) {
         reqBody.mtdInterval = Number(this.parameters.mtdInterval);
       }
+      if (this.parameters.networkSizeList) {
+        reqBody.networkSizeList = Number(this.parameters.networkSizeList);
+      }
+      if (this.parameters.scheme) {
+        reqBody.scheme = Number(this.parameters.scheme);
+      }
```
Comment (FearMyCode) [inline client/src/stores/simulation.js:29]: Change this back to this.parameters.scheme.toLowerCase(). Scheme is not a number

**Review by FearMyCode — CHANGES_REQUESTED** [review]
You might also consider looping through the parameters property. It can enhance readability and scalability.

File: client/src/stores/simulation.js (hunk 1 above)
Comment (crystalycteh) [reply client/src/stores/simulation.js]: ok

**Comment by benvarian** [comment]
if this is ready to be looked at again, fix merge conflicts and let someone know

**Pull request closed without merging**